src/Pages/CategoriePage/ItemCategory.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import './ItemCategory.scss';
+import ShirtList from './ShirtList';
+
+class ItemCategory extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      click: false,
+      imgChange: true,
+      harucheck: true,
+    };
+  }
+
+  upDown = () => {
+    const { click } = this.state;
+    this.setState({ click: !click });
+  };
+
+  render() {
+    const { click } = this.state;
+    const { handleItemFilter, handleSaleFilter } = this.props;
+    const upDownImg = click ? '../../../images/up.png' : '../../../images/down.png';
+    return (
+      <div className="ItemCategory">
+        <nav className="nav">
+          <div className="navBox">
+            <div className="mainCategory">
+              <div className="categoryTitle">상품옵션</div>
+              <div className="categoryList">
+                <input type="checkbox" onChange={handleItemFilter} />
+                <img src="./images/haru2.png" alt="하루배송 사진" />
+              </div>
+              <div className="categoryList">
+                <input type="checkbox" onChange={handleSaleFilter} />
+                <span>세일</span>
+              </div>
+            </div>
+          </div>
+          <div className="categoryBox">
+            <div className="category">CATEGORY</div>
+            <div className="categoryList">전체</div>
+            <div className="categoryOuterBox">
+              <div className="categoryListSub">아우터</div>
+              <div>
+                <img className="arrow" src="./images/down.png" alt="down" />
+              </div>
+            </div>
+            <div className="categoryOuterBox" onClick={() => this.upDown()}>
+              <div className="categoryListSub">상의</div>
+              <div>
+                <img className="arrow" src={upDownImg} alt="down" />
```

Comment:
- render 될 때마다 변수를 선언시키기 보다는 여기서 바로 삼항연산자로 처리해주는 것이 좋습니다. 이미지도 파일 하나에서 변수로 전부 관리하고 import 해서 사용하면 더 좋지 않을까요? :)
- nesting 사용하면 위의 div에 className 주는 것도 좋을 것 같습니다.
- alt값 부여해주세요.
```suggestion
                <img alt="arrow' src={click ? upArrow : downArrow} />
```